### PR TITLE
Add arcp 

### PR DESCRIPTION
**Pull request by pvanheus**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by pvanheus** [comment]
I am willing to be listed as a maintainer for this package.

File: recipes/arcp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{% set name = "arcp" %}
+{% set version = "0.2.0" %}
+{% set sha256 = "ad9df671a6cacfb44e85cd2971eee87e663a5bbf84618b31476b9b452bb01a41" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/stain/arcp-py/releases/download/0.2.0/arcp-{{ version }}.tar.gz
```
Comment (xhochy) [inline recipes/arcp/meta.yaml:10]:
```suggestion
  url: https://github.com/stain/arcp-py/releases/download/{{ version }}/arcp-{{ version }}.tar.gz
```

File: recipes/example/meta.yaml [hunk 2]
```diff
@@ -1,83 +0,0 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (chrisburr) [inline recipes/example/meta.yaml:1]: Can you restore the example, the CI will ignore any recipes which aren't part of your PR.

File: recipes/arcp/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  sha256: {{ sha256 }}
```
Comment (chrisburr) [inline recipes/arcp/meta.yaml:11]: As it's only used once there is no need to use a variable:
```suggestion
  sha256: ad9df671a6cacfb44e85cd2971eee87e663a5bbf84618b31476b9b452bb01a41
```

File: recipes/arcp/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
+{% set name = "arcp" %}
+{% set version = "0.2.0" %}
+{% set sha256 = "ad9df671a6cacfb44e85cd2971eee87e663a5bbf84618b31476b9b452bb01a41" %}
```
Comment (chrisburr) [inline recipes/arcp/meta.yaml:3]:
```suggestion
```

**Comment by pvanheus** [comment]
@chrisburr I normally put the sha256 variable at the top of recipes I write because I like having bits that change often in a block at the top but I have merged your suggestions since they align with the canonical example recipe.

File: recipes/example/meta.yaml (hunk 2 above)
Comment (pvanheus) [reply recipes/example/meta.yaml]: Done.

**Comment by npavlovikj** [comment]
@pvanheus , I think the download link should be `https://github.com/stain/arcp-py/archive/0.2.0.tar.gz` with sha256 a03c1ec28781c6a50704bd776439d38f5393c45d71385583be584346b646d688.
This tarball also has the license file in it, so you can remove LICENSE.txt from this PR.

**Comment by npavlovikj** [comment]
Thank you @pvanheus  🎉

**Pull request merged**